### ISSUE TITLE
redis6 name causing confusion

### ISSUE THREAD

**Issue by sumitgarg44** [iusrepo/redis6#4]
Package name is redis6, though it provides version 6.2.X which is different from 6.0.X.

Would it be better to rename repo name and package name in spec to redis62, considering few must have already installed it with previous name redis6? 

I think of the following actions to complete rename:

1. Update package name in spec file from redis6 to redis62
2. Update repo name
3. Remove old redis packages from repo, once redis62 is published.

If community thinks alike, I can carry out the step1 but would need help for step2 & 3 due to access restrictions.

**Comment by carlwgeorge** [comment]
The only reason anyone would expect it to stay locked to 6.0.x is if it were named redis60.  As it's named redis6, it should be expected to track 6.x.x, not 6.0.x.  IUS started naming redis packages after the major version only when it became clear the upstream was doing branches based off the major version (4.0, 5.0, etc).  The 6.2 branch was a bit of a curveball.  The [6.2 release notes](https://github.com/redis/redis/blob/6.2.0/00-RELEASENOTES#L416-L426) do claim that it's mostly a superset of 6.0 features.  I understand the desire to rename the packages, but I strongly advise against it because I've never seen a clean way to keep the unit file enabled when it moves between packages.  An update should not cause a previously enabled unit file to be disabled.  We looked at this previously when we stopped naming packages with a "u" suffix, and at the time we decided it wasn't worth it and left existing package names alone.
